.github/FUNDING.yml

```diff
@@ -1 +1,5 @@
-github: ota-meshi
+# These are supported funding model platforms
+
+github: [ota-meshi, JounQin]
+open_collective: rxts
+patreon: 1stG
```

Comment:
Thank you for this PR!
Are you interested in maintaining this project together?
If so, I invite you to this project.

Reply:
Let's do it!

Reply:
I will invite you to projects to check JSON(C) with ESLint!

Reply:
I invited you to the project on GitHub and npm. Please let me know if anything is leaking.